server/room.mjs

```diff
@@ -66,7 +65,7 @@ export default class Room {
     } catch(e) {
       Logging.log(`ERROR LOADING FILE: ${e.toString()}`);
       try {
-        fs.writeFileSync(path.resolve() + '/save/errors/' + Math.random().toString(36).substring(3, 7), src);
+        fs.writeFileSync(Config.directory('save') + '/errors/' + Math.random().toString(36).substring(3, 7), src);
```

Comment:
Not sure why errors get saved to a random filename, but this PR is not changing that behavior.